components/nexus-core/src/main/java/org/sonatype/nexus/internal/security/StaticSecurityConfigurationResourceImpl.groovy

```diff
@@ -909,16 +909,19 @@ class StaticSecurityConfigurationResourceImpl
                     'permission': 'security:usersforgotid'
                 ]
             ),
-            new CPrivilege(
-                id: 'usersresetpw', // 59
-                type: 'method',
-                name: 'User Reset Password - (delete,read)',
-                description: 'Give permission to reset any user\'s password.',
-                properties: [
-                    'method'    : 'delete,read',
-                    'permission': 'security:usersreset'
-                ]
-            ),
+
+            // NOTE: This privilege is no longer used by anything
```

Comment:
I left here just to keep record of what 'id: 59' was.

I expect we'll need to overhaul the permissions at some time in the future to normalize namespace, etc and we can remove it then.